### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by shagedorn**

### CODE REVIEW

File: ISHLogDNA.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -166,10 +217,14 @@
 		DC0C28071EF0764500B27023 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastSwiftUpdateCheck = 0830;
+				LastSwiftUpdateCheck = 0900;
```
Comment (mollidor) [inline ISHLogDNA.xcodeproj/project.pbxproj]: That cutting edge?

File: ISHLogDNA.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -166,10 +217,14 @@
...
 				LastUpgradeCheck = 0900;
 				ORGANIZATIONNAME = iosphere;
 				TargetAttributes = {
+					1BE21AF01F44336A0055FAF2 = {
+						CreatedOnToolsVersion = 9.0;
```
Comment (mollidor) [inline ISHLogDNA.xcodeproj/project.pbxproj]: same here. I'm fine with it. Just highlighting.

File: ISHLogDNA.xcodeproj/project.pbxproj (hunk 1 above)
Comment (shagedorn) [inline ISHLogDNA.xcodeproj/project.pbxproj]: Works fine with Xcode 8.

Also, this only affects sample projects and tests. We don't provide a framework project, just two source files.

**Comment by mollidor** [comment]
👍

File: ISHLogDNA.xcodeproj/project.pbxproj (hunk 1 above)
Comment (shagedorn) [inline ISHLogDNA.xcodeproj/project.pbxproj]: Sorry, not actually true (although this line is not the issue). Creating the target in Xcode 9 made it use Swift 4 and deployment target iOS 11, which are not available in Xcode 8.

Changes those build settings (no code changes required). Now builds (and tests succeed) in Xcode 8 and 9, without warnings, except for Xcode 9 prompting about Swift 4 migration.

Comment (mollidor) [inline ISHLogDNA.xcodeproj/project.pbxproj]: Sorry. Should have been more precise. That was my concern.

**Pull request merged**